controllers/v1/entityTypes.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * name : entityTypes.js
+ * author : Rakesh Kumar
+ * created-date : 19-March-2020
+ * Description : Entity types information. 
+ */
+
+// Dependencies
+
+const entityTypesHelper = require(MODULES_BASE_PATH + "/entity-types/helper.js");
+//  const bulkUploadHelper = require(MODULES_BASE_PATH + "/bulk-upload-request/helper.js");
```

Comment:
@rakeshSgr - Remove uncommented code if not required.

Reply:
resolved